Utilities/XcodeBuildScripts/openssl.bash

```diff
@@ -95,6 +95,8 @@ for BUILD_ARCH in $BUILD_ARCHS; do
 
     if [ "$BUILD_ARCH" == "x86_64" ]; then
         ./Configure darwin64-x86_64-cc shared enable-ec_nistp_64_gcc_128 no-ssl2 no-ssl3 no-comp no-async --prefix="$OPENSSL_TMP/" --openssldir="$OPENSSL_TMP/"
+    elif [[ "$BUILDARCH" = *"i386"* ]]; then
```

Comment:
Let's do `== "i386"` rather than `= *"i386"*`, since we know it should be that string exactly?